CRM/Core/BAO/Website.php

```diff
@@ -57,6 +56,31 @@ public static function add(&$params) {
     return $website;
   }
 
+  /**
+   * Create website.
+   *
+   * If called in a legacy manner this, temporarily, fails back to calling the legacy function.
+   *
+   * @param array $params
+   * @param int $contactID
+   * @param bool $skipDelete
+   *
+   * @return bool|CRM_Core_BAO_Website
+   */
+  public static function create($params, $contactID = NULL, $skipDelete = NULL) {
+    if ($skipDelete || ($contactID && !is_array($contactID))) {
```

Comment:
What about being more specific instead of checking `$skipDelete` for truthyness check if it is not null?

Reply:
on the off possibility someone was passing in FALSE in the past? I guess I can - although I changed the only places in core & suspect that it's not being called from outside of core

Reply:
OK - done